components/getStarted/GetStartedCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { Card, Button, Row } from "react-bootstrap";
+import Image from "../Image";
+
+export interface GetStartedCardProps {
```

Comment:
Oh, we'll want an `href` property since one link goes to docs and the other goes to `/trial`